src/applications/pensions/config/chapters/03-health-and-employment-information/helpers.js

```diff
@@ -75,15 +76,17 @@ MedicalCenterView.propTypes = {
 export const generateMedicalCentersSchemas = (
   medicalCentersKey = 'medicalCenters',
   medicalCentersTitle = 'Default Medical Centers Title',
-  medicalCenterMessage = 'Default Message',
+  medicalCenterMessage = 'Default Message', // THIS ONE
```

Comment:
Do we want to remove this inline comment?

Reply:
Oops - thanks for catching that!